uninstall.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Distributor uninstall script.
+ *
+ * @since x.x.x
+ * @package distributor
+ */
+
+// phpcs:disable
+
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	die;
+}
+
+global $wpdb;
+
+// Delete options.
+$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name LIKE 'dt\_%';" );
+
+// Remove transients.
+$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name LIKE '_transients\_dt\_%';" );
```

Comment:
* this needs to account for the transient without a key https://github.com/10up/distributor/blob/4ceb996dc23e6cdec514bb53544998ea922d8e8d/includes/classes/InternalConnections/NetworkSiteConnection.php#L864-L865
* we might need to flush the option's and all_options caches for this. Transients are only stored in the options table for non-persistent caches, so it the page immediately refreshes after this then we might be able to get away with it

Reply:
@peterwilsoncc I prefixed it. Let me know if I need to include any other transient key.